Runtime/Model/Breadcrumbs/IBacktraceBreadcrumbs.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Backtrace.Unity.Model.Breadcrumbs
+{
+    public interface IBacktraceBreadcrumbs
+    {
+        BacktraceBreadcrumbType BreadcrumbsLevel { get; }
+        bool EnableBreadcrumbs(BacktraceBreadcrumbType level, UnityEngineLogLevel unityLogLevel);
+        bool ClearBreadcrumbs();
+        bool Log(string message, LogType type, IDictionary<string, string> attributes);
+        bool Log(string message, LogType type);
+        bool Debug(string message);
+        bool Debug(string message, IDictionary<string, string> attributes);

```

Comment:
no, because is Log is the universal method if someone is learning how our API works. Debug, Warning, Error etc are methods that I grabbed from popular logging frameworks and I think we expecting that users will be by default familiar with this API.